src/datasets/features/audio.py

```diff
@@ -309,11 +315,16 @@ def _decode_mp3(self, path_or_file):
                         "To support 'mp3' decoding with `torchaudio>=0.12.0`, please install `ffmpeg4` system package. On Google Colab you can run:\n\n"
                         "\t!add-apt-repository -y ppa:jonathonf/ffmpeg-4 && apt update && apt install -y ffmpeg\n\n"
                         "and restart your runtime. Alternatively, you can downgrade `torchaudio`:\n\n"
-                        "\tpip install \"torchaudio<0.12\"`.\n\nOtherwise 'mp3' files will be decoded with `librosa`:\n\n"
+                        "\tpip install \"torchaudio<0.12\".\n\nTo decode 'mp3' files without torchaudio, please install `librosa`:\n\n"
                         "\tpip install librosa\n\nNote that decoding will be extremely slow in that case."
                     ) from err
                 # try to decode with librosa for torchaudio>=0.12.0 as a workaround
-                warnings.warn("Decoding mp3 with `librosa` instead of `torchaudio`, decoding is slow.")
+                global _librosa_warned
+                if not _librosa_warned:
+                    warnings.warn(
+                        "Decoding mp3 with `librosa` and `audioread` instead of `torchaudio`, decoding is slow."
```

Comment:
and in that case we don;t need to mention audioread here. If the user has a working `libsndfile` setup then it doesn't use audioread anyway
```suggestion
                        "Decoding mp3 with `librosa` instead of `torchaudio`, decoding is slow."
```

Reply:
hm true! probably mac and windows users do have a working setup if they have the latest version of `pysoundfile`, didn't think about that (being too linux-centric apparently...)